group_project_v2/project_api/api_implementation.py

```diff
@@ -19,39 +16,18 @@
 SUBMISSION_API = '/'.join([API_PREFIX, 'submissions'])
 GROUP_API = '/'.join([API_PREFIX, 'groups'])
 COURSES_API = '/'.join([API_PREFIX, 'courses'])
+PROJECTS_API = '/'.join([API_PREFIX, 'projects'])
 
 
-DEFAULT_EXPIRATION_TIME = timedelta(seconds=10)
-
+def do_not_cache(reason):  # pylint:disable=unused-argument
```

Comment:
I agree. At first I thought this was something like the `django.views.decorators.cache.never_cache` decorator, which does have an effect. I think a simple comment would be more clear.